src/Model/helpers.js

```diff
@@ -21,3 +22,20 @@ export const createTimestampsFor = (model: Model) => {
 
   return timestamps
 }
+
+export async function fetchChildren(model: Model): Promise<Model[]> {
+  const hasManyAssociations = Object.entries(model.constructor.associations).filter(([, value]) => value.type === 'has_many')
+  const childrenQueries = hasManyAssociations.map(([key, value]) => {
+    const childCollection = model.collections.get(key)
+    return childCollection.query(Q.where(value.foreignKey, model.id))
+  })
```

Comment:
can you split this function into smaller, easier to understand pieces? e.g. `getChildrenQueries(Class<Model>) → Query<Model>[]`

Reply:
done